### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by EVogel1999**
This is a UI redesign of security injections.  The aim of this is to make the modules more modern and fitting with the current cyber4all team.  This design makes the module more modifiable, UI speaking, allowing for more, or less, sections then the standard 6.
image
Mobile View:
image
image

### CODE REVIEW

File: resources/template.html [hunk 1]
```diff
@@ -1,91 +1,73 @@
...
-	<div ng-controller="modCtrl" onselectstart="return false" autocomplete=off>
-		<header class="header">
-			<div class="container">
-				<div class="row">
-					<div class="col-xs-10">
-						<h1 class="header-text">
+	<main ng-controller="modCtrl" onselectstart="return false" autocomplete=off>
+		<div id="side-tabs">
+			<a class="feedback" data-toggle="modal" data-target="#fbModal">
+				<img alt="Feedback" src="http://cis1.towson.edu/~cssecinj/wp-content/uploads//feedback9.png">
```
Comment (pzalep1) [inline resources/template.html]: alt="Submit Feedback"

File: resources/template.html [hunk 2]
```diff
@@ -232,9 +237,6 @@ <h5 ng-bind-html="unit.prompt"></h5>
 										<p>Complete by clicking the corresponding code below</p>
 										<pre ng-if="unit.code" class="si-checklist-code" ng-bind-html="unit.code"></pre>
 										<script ng-if="unit.javascript" ng-bind="unit.javascript"></script>
-										<noscript>
-											<!--NEED SOMETHING HERE-->
```
Comment (pzalep1) [inline resources/template.html:236]: Should be a message for browsers that don't support javascript

File: resources/template.html [hunk 3]
```diff
@@ -323,68 +314,31 @@ <h5>Congratulations! You have completed the {{ module.variant }} {{ module.name
 								<div id="si-certificate-pane" style="display:none">
 									<b>You must email the certificate to your instructor to obtain credit</b>
 									<br>
-									<iframe title="Certificate" alt="Certificate" width="auto" height="348" ng-src="{{detailFrame}}"></iframe>
+									<iframe title="Certificate" alt="Certificate" width="auto" height="348"
+										ng-src="{{detailFrame}}"></iframe>
 									<br>
-									<a class="btn btn-info btn-md" id="si-certificate-link" download="certificate.pdf">Download
+									<a class="btn btn-info btn-md" id="si-certificate-link"
```
Comment (pzalep1) [inline resources/template.html]: Should be a button. Links go to new pages, buttons perform actions on pages

File: resources/template.html [hunk 4]
```diff
@@ -399,35 +353,42 @@ <h4 class="modal-title" id="fbModalLabel">Feedback</h4>
...
 								<br>
 							</div>
-							<div vc-recaptcha theme="'light'" key="model.key" on-create="setWidgetId(widgetId)" on-success="setResponse(response)"></div>
+							<div vc-recaptcha theme="'light'" key="model.key" on-create="setWidgetId(widgetId)"
+								on-success="setResponse(response)"></div>
 							<p ng-class="result" style="padding: 15px; margin: 0;">{{ resultMessage }}</p>
 						</div>
 						<div class="modal-footer">
 							<button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
-							<button type="submit" class="btn btn-primary" ng-disabled="submitButtonDisabled">Submit</button>
```
Comment (pzalep1) [inline resources/template.html:430]: aria-label

File: resources/template.html [hunk 5]
```diff
@@ -399,35 +353,42 @@ <h4 class="modal-title" id="fbModalLabel">Feedback</h4>
 						<div class="modal-body">
 							<h5>Use the form below to send us comments or report problems with specific modules</h5>
 							<h5>Alternatively, email us at
-								<a href="mailto:securityinjections@towson.edu?subject=Feedback">securityinjections@towson.edu</a>
+								<a
```
Comment (pzalep1) [inline resources/template.html]: Link needs aria-label

**Review by pzalep1 — CHANGES_REQUESTED** [review]

File: resources/template.html [hunk 6]
```diff
@@ -1,91 +1,73 @@
 <div ng-app="modApp">
-	<div ng-controller="modCtrl" onselectstart="return false" autocomplete=off>
-		<header class="header">
-			<div class="container">
-				<div class="row">
-					<div class="col-xs-10">
-						<h1 class="header-text">
+	<main ng-controller="modCtrl" onselectstart="return false" autocomplete=off>
+		<div id="side-tabs">
+			<a class="feedback" data-toggle="modal" data-target="#fbModal">
```
Comment (pzalep1) [inline resources/template.html:4]: Why is the link here twice?

Comment (EVogel1999) [reply resources/template.html:4]: No idea.  I just removed it.

**Pull request merged**